### PR TITLE
MIBI-add acquisition instrument vendor/model

### PR DESCRIPTION
**Pull request by cebriggs7135**
Added acquisition instrument vendor and model to the level 2 metadata for MIBI schema.
@ngehlenborg These 2 fields are required for all assays but they were omitted here so I'm adding them as we finish up the docs for MIBI. These should actually be included in level 1 metadata.

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/assays/mibi-v1.yaml
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/assays/mibi-v1.yaml:40]:
```suggestion
```
Defaults do not need to be given

File: src/ingest_validation_tools/table-schemas/assays/mibi-v1.yaml
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/assays/mibi-v1.yaml:50]:
```suggestion
```

**Review by mccalluc — CHANGES_REQUESTED** [review]
- Please remove defaults
- Will wait for confirmation from Nils that this is ok to modify in place

**Review by ngehlenborg — APPROVED** [review]
I approve assuming  "modify in place" = "modify without bumping the version"

**Comment by mccalluc** [comment]
I generate the docs here. Thanks.

**Comment by mccalluc** [comment]
@cebriggs7135 -- Could you check the placement of these fields in the list? I believe that for other assays they are at the top of the "Unique to this type" list... but that's not the case here. Should they be?

**Comment by cebriggs7135** [comment]
@mccalluc You are correct. Acquisition instrument fields should be at the top of the "Unique to this type" list.

**Comment by mccalluc** [comment]
@cebriggs7135 -- Ok, good to know: They were misplaced at the start of this PR. If you can review the output now and confirm that the field order is what you want, I'd appreciate it. Will wait to merge until I get your confirmation.

**Review by cebriggs7135 — COMMENTED** [review]
looks correct. thanks.

**Comment by mccalluc** [comment]
Converting back to draft. This is not ready to go. The changes in https://github.com/hubmapconsortium/ingest-validation-tools/pull/1034 should be merged into this.

**Comment by mccalluc** [comment]
Close in favor of #1035

**Pull request closed without merging**
